layers/+completion/spacemacs-helm/config.el

```diff
@@ -0,0 +1,14 @@
+;;; config.el --- Spacemacs-helm Layer config File
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defvar spacemacs-helm-basic-config nil
+  "If non nil, don't bind keys globally in spacemacs-helm and
+don't enable helm-mode.")
```

Comment:
Actually the local package is not a good solution.

Consider creating a new layer `spacemacs-helm-basic` which is automatically added by `spacemacs-helm`. This way we keep the isolation and the user can choose between a minimal helm config and a the full blown config.

Mmmmmhh I wonder if `spacemacs-ivy` should automatically include `spacemacs-helm-basic` as well. Would make sense to me.